src/sass/base/_mixins.sass

```
@@ -0,0 +1,63 @@
+@mixin flex
+    display: flex
+    align-items: center
+    justify-content: center
```

Cho truyền vào các biến luôn chứ nhỉ.
```suggestion
@mixin flex($justify-content: center, $align-items: center)
    display: flex
    align-items: $align-items
    justify-content: $justify-content
```
Ngoài ra có thể bổ sung thêm nhiều thứ khác như flex-wrap, flex-direction, gap,...